src/common/settings_gen.ts

```diff
@@ -1,431 +1,458 @@
 import { JSONSchema7 } from 'json-schema';
```

Comment:
Re-generated this file with `cargo run > ../../src/common/settings_gen.ts`. Are you running a lint configuration on it?